modules/cluster/addons.tf

```diff
@@ -25,28 +25,26 @@ module "critical_addons_node_group" {
 
 data "aws_region" "current" {}
 
-# The kube-proxy EKS addon introduced regressions of #124 and #209. We will move to the EKS addon when these are fixed.
-module "kube_proxy" {
-  source = "./kubectl"
-  config = local.config
-  manifest = templatefile(
-    "${path.module}/addons/kube-proxy.yaml",
-    { aws_region = data.aws_region.current.name },
-  )
-}
 // When upgrading k8s version run `aws eks describe-addon-versions --kubernetes-version <version>` to get addon_version numbers
 
 resource "aws_eks_addon" "vpc-cni" {
   cluster_name      = local.config.name
   addon_name        = "vpc-cni"
-  addon_version     = "v1.10.2-eksbuild.1"
+  addon_version     = "v1.11.0-eksbuild.1"
+  resolve_conflicts = "OVERWRITE"
+}
+
+resource "aws_eks_addon" "kube-proxy" {
+  cluster_name      = local.config.name
+  addon_name        = "kube-proxy"
+  addon_version     = "v1.21.2-eksbuild.2"
```

Comment:
The docs recommend `1.22.6-eksbuild.1` https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html

Reply:
Thx! fixed